packages/compiler/test/core/compiler-code-fixes/model-to-literal.codefix.test.ts

```diff
@@ -21,3 +21,21 @@ it("it change model expression to an object value", async () => {
       }
     `);
 });
+
+it("it recursively changes the model expression to the corresponding object value", async () => {
+  await expectCodeFixOnAst(
+    `
+      @example(┆{Bar : {Baz : "Hello"}})
```

Comment:
what if the top object need conversion, the 2nd layer object doesn't need conversion, and the 3rd layer object need conversion?